src/moditect/module-info.java

```diff
@@ -0,0 +1,19 @@
+module com.fasterxml.jackson.core {
```

Comment:
I feel your pain. Note ModiTect would allow you to use wildcards (com.fasterxml.jackson.core.*) if you were to use it on JDK 9 or later. It will then generate the actual descriptor exporting all packages matching the wildcard.

Reply:
Ok that is good to know going forward. While Jackson 2.x will probably not go beyond JDK 8, 3.x will at some point so that's good info.